cli/responseemitter_test.go

```diff
@@ -2,10 +2,11 @@ package cli
 
 import (
 	"bytes"
+	"fmt"
 	//"io"
 	"testing"
 
-	"github.com/ipfs/go-ipfs-cmdkit"
+	//"github.com/ipfs/go-ipfs-cmdkit"
```

Comment:
Should probably remove before final merge.

Reply:
yup.